File: webTrain/Model/Station.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;
using System;


namespace webTrain.Model
{
    public class Station
    {
        [BsonRepresentation(BsonType.String)]
        public string nameStation { get; set; }

        [BsonRepresentation(BsonType.DateTime)]
        public DateTime? arrivalTime { get; set; }

        [BsonRepresentation(BsonType.DateTime)]
        public DateTime? departureTime { get; set; }

        public Station()
        {
           
        }

        public Station(string nameStation, DateTime? arrivalTime, DateTime? departureTime)
        {
            this.nameStation = nameStation;
            this.arrivalTime = arrivalTime;
            this.departureTime = departureTime;
        }

    }
}

File: webTrain/ClientApp/src/app/Models/station.ts
export class Station {
    nameStation: string;
    arrivalTime: Date;
    departureTime: Date;
    constructor(_nameStation: string, _arrivalTime: Date, _departureTime: Date) {
        this.nameStation = _nameStation;
        this.arrivalTime = _arrivalTime;
        this.departureTime = _departureTime;
    }
}
